components/layout/Container.js

```diff
@@ -10,8 +10,10 @@ export default function Container ({
   const commonClassNames = `${flex ? `${cssBreakingPoint}:flex` : 'block'} ${flexReverse ? `${cssBreakingPoint}:flex-row-reverse` : ''} ${className} px-4 sm:px-6 lg:px-8`
   const wideClassNames = `max-w-screen-xl ${commonClassNames}`
   const regularClassNames = `max-w-4xl ${commonClassNames}`
-  const normalClassNames = `${wide ? wideClassNames : regularClassNames} mx-auto`
-  const fluidClassNames = `${commonClassNames}`
+  const normalClassNames = `${commonClassNames}`
+  const fluidClassNames = `${wide ? wideClassNames : regularClassNames} mx-auto`
```

Comment:
What has changed here besides the order of the lines? Additionally, on the home page everything is stretched (and that can't be):

![image](https://user-images.githubusercontent.com/20404945/163968962-4f83b480-c718-4085-b4d8-8463698703e2.png)